src/lib/guesstimator/formatter/formatters/DistributionPointText.js

```diff
@@ -10,7 +10,7 @@ export const item = Object.assign(
     format(g) {
       const {guesstimateType} = this
       const value = parseNumber(g.text)
-      return {guesstimateType, value}
```

Comment:
No, it renders as 0. I modified something else that makes it strip trailing zeros from the representation in all cases.